### PR TITLE
SW-5634 Allow sections to be assigned an owner

### PR DESCRIPTION
**Pull request by constanzauanini**
Screenshot 2024-07-18 at 9 50 22 AM
Screenshot 2024-07-18 at 9 50 40 AM

### CODE REVIEW

**Comment by joshbuchea** [comment]
Apologies for not reviewing this sooner, yesterday I was hyper focus on trying to wrap up SW-5638 (FE - Add document outline) 🙇  Reviewing now 👀

File: src/components/DocumentProducer/MultiLineComponentNonEditable/index.tsx [hunk 1]
```diff
@@ -19,23 +43,158 @@ export default function MultiLineComponentNonEditable({
...
+      reload();
+    }
+  }, [updateWorkflowDetailsResponse]);
+
+  useEffect(() => {
+    if (updateOwnerResponse?.status === 'success') {
+      snackbar.toastSuccess(strings.SECTION_OWNER_ASSIGNED);
+      reload();
+    }
+  }, [updateOwnerResponse]);
```
Comment (joshbuchea) [inline src/components/DocumentProducer/MultiLineComponentNonEditable/index.tsx:84]: Should we provide feedback for these requests when the response is unsuccessful?

**Review by joshbuchea — APPROVED** [review]

**Comment by joshbuchea** [comment]
Just one question, changes look good though. Nice work!

File: src/components/DocumentProducer/MultiLineComponentNonEditable/index.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/DocumentProducer/MultiLineComponentNonEditable/index.tsx:84]: Yes, probably show the generic error. Will update it

**Pull request merged**